packages/target-tools/src/telemetryProvider.spec.js

```diff
@@ -5,19 +5,28 @@ describe("TelemetryProvider", () => {
     requestId: "123456"
   };
 
+  const TARGET_NOTIFICATION_REQUEST = {
+    notifications: []
+  };
+
   const TARGET_TELEMETRY_ENTRY = {
     execution: 1
   };
 
   it("adds an entry", () => {
```

Comment:
Let's rename test something like "adds and executes entries" since it's more of an E2E test for the TelemetryProvider.  Let's also call `addEntry` 3x times before calling `executeTelemetries`.  Sometimes strange bugs present themselves when doing something more than once.